docs/UserGuide.md

```diff
@@ -54,7 +54,7 @@ This user guide uses various formatting styles to facilitate reading and to comm
 ### 2.2 Trying out
 
 1. Let's try out your first command by adding a new property! Type <br>`add property n/Bishan t/Hdb a/Blk 150 Bishan Street 11 #02-101 p/570150 d/30-6-2021` into the command box and press <kbd>Enter</kbd> to execute it. After that, scroll down to the bottom of the property list to see your newly added property.
-1. Want to view the properties in order of their asking price? Type the command <br>`sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be ordered in descending order, with the property having the highest asking price at the top.
+1. Want to view the properties in order of their asking price? Type the command <br>`sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be in descending order based on price.
```

Comment:
I think can keep the part act the first property having highest price at the book though, although the phrasing for the first half is good now

Reply:
Shall update this